plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/ISharedObjectsTransformer.java

```diff
@@ -26,10 +26,13 @@
 import org.pentaho.di.shared.SharedObjects;
 
 public interface ISharedObjectsTransformer extends ITransformer {
+
+  @Deprecated
```

Comment:
why is it deprecated. Can you add a comment

Reply:
added.